test/jdk/java/net/DatagramSocket/DatagramSocketMulticasting.java

```diff
@@ -57,10 +59,19 @@
 import static java.net.StandardSocketOptions.SO_REUSEADDR;
 import static jdk.test.lib.NetworkConfiguration.isSameInterface;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class DatagramSocketMulticasting {
     static final ProtocolFamily UNSPEC = () -> "UNSPEC";
+    static final int MAX_TRIES = 3;
 
-    public static void main(String[] args) throws IOException {
+    @Test
+    public void main() throws IOException {
         IPSupport.throwSkippedExceptionIfNonOperational();
```

Comment:
This can throw jtreg.SkippedException, do you know if the JUnit test runner handles this?

Reply:
Good point, I'll look into that

Reply:
There's no special handling for such an exception type in place.

Could the "if non operational" state be passed to a call to an `Assumptions.assume...()` method instead of throwing?
- https://junit.org/junit5/docs/current/user-guide/#writing-tests-assumptions

Other alternatives are:
- https://junit.org/junit5/docs/current/user-guide/#writing-tests-conditional-execution
- https://junit.org/junit5/docs/current/user-guide/#extensions-conditions

Reply:
Thanks for checking. In that case, I suspect it will need to change to
`assumeTrue(IPSupport.currentConfigurationIsValid(), "Invalid networking configuration");`